### DISCUSSION TITLE
組織名について

### DISCUSSION THREAD

**Discussion by intsuc** [RuskEocssar/MC-Command-Note#18] [General]
今更なのですが、[technology sharesは技術系の株という意味が強い](https://www.google.com/search?q=%22Technology+Shares%22)と思うので、改名しませんか？

**Comment by RuskEocssar** [comment]
ほんとにそうなんですよね 代わりの名前思いつかなくて放置してるだけなのでいい加減変えたいんですが、なにがいいんでしょうかね...

**Reply by RuskEocssar** [reply]
Wikiっぽい運用がメインになったのでTechnical wikiが唯一の案です

**Reply by intsuc** [reply]
○○ Collectiveも良さそうだと思います。

> Collectiveとは、少なくとも1つの共通の問題や関心事を共有する、あるいはその動機となる主体の集まりであり、共通の目的を達成するために協力するものである<sup>[要出典]</sup>。Collectivesは、必ずしも経済的利益や節約に重点を置いているわけではないという点でCooperativeとは異なるが、そのような場合もある。

**Reply by RuskEocssar** [reply]
共同体って感じの表現はいいですね！
既存の表現でTech Community[[1]](https://techcommunity.microsoft.com/)というのもあって、個人的にはこれもありな気がしました。

**Reply by intsuc** [reply]
communityも良さそうですね。調べてみるとcommunityはcollectiveより（範囲や責任などが）緩いみたいなので、今の形態にはcommunityが適しているかもしれません。
